src/sidebar/app.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
```

Comment:
My first instinct is to not include `build-html` with `build`. I guess that means this file gets produced for every build now and it's a little unnecessary for anything other than a custom build correct?

 Its confusing because our own app.html is owned by h./ and this file is really only used for ad-hoc hosting of the a forked client.

I'm not really giving direction here, but just speaking to the larger picture.  Here is kind of what I was thinking....

```
const buildCustomAssets = gulp.parallel(
  'build-js',
  'build-css',
  'build-fonts',
  'build-html',
  'build-images'
);
const buildAssets = gulp.parallel(
  'build-js',
  'build-css',
  'build-fonts',
  'build-images'
);
gulp.task('build', gulp.series(buildAssets, generateManifest));
gulp.task('build-custom', gulp.series(buildAssets, generateManifest));
```

And then the script calls `build-custom` for example... in my mind, then the /build dir would more accurately reflect what was run rather than being a swiss army know output.